org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/entities/EntitiesDocumentationUtils.java

```diff
@@ -64,6 +65,19 @@ public String getValue() {
 	private EntitiesDocumentationUtils() {
 	}
 
+	public static MarkupContent getDocumentation(String entityName, String entityValue, EntityOriginType type,
+			boolean markdown) {
+		return getDocumentation(entityName, entityValue, null, null, null, type, markdown);
+	}
+
+	public static MarkupContent getDocumentation(DTDEntityDecl entity, EntityOriginType type, boolean markdown) {
+		String systemID = entity.getSystemId();
+		String publicID = entity.getPublicId();
+		String targetURI = entity.getNameParameter().getTargetURI();

```

Comment:
Do we need to check if  `entity.getNameParameter() == null` here? Although, I'm not sure if it would happen in our case

Reply:
If entity has name, entity.getNameParameter() is not null, so we don't need to check that.